pkgs/development/libraries/pdal/default.nix

```
@@ -0,0 +1,101 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  fetchpatch,
+
+  cmake,
+  pkgconfig,
```

canonical name > alias
```suggestion
  pkg-config,
```